### PR TITLE
Foundations: understanding errors list style

### PR DESCRIPTION
**Pull request by damon314159**
## Because
The list style and markdown linting had not been applied to this lesson file

## This PR
Makes the required changes to this lesson file

## Issue

Contributes to #27119

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/javascript_basics/understanding_errors.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Reading and understanding error messages is a requirement as a developer. At fir
 
 This section contains a general overview of topics that you will learn in this lesson.
 
-- Name at least three kinds of Javascript errors
+- Name at least three kinds of JavaScript errors
```
Comment (JoshDevHub) [inline foundations/javascript_basics/understanding_errors.md:9]: This group of list items should have periods added to the end.

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Looks great. Just one small nit, and then I'll approve/merge.

**Comment by damon314159** [comment]
@JoshDevHub that one slipped past me - should be good to go now.

**Review by JoshDevHub — APPROVED** [review]
:rocket:

**Pull request merged**